node/actors/network/src/gossip/mod.rs

```diff
@@ -12,16 +12,22 @@
 //! Static connections constitute a rigid "backbone" of the gossip network, which is insensitive to
 //! eclipse attack. Dynamic connections are supposed to improve the properties of the gossip
 //! network graph (minimize its diameter, increase connectedness).
+pub use self::attestation_status::{
+    AttestationStatusClient, AttestationStatusReceiver, LocalAttestationStatus,
+};
 pub use self::batch_votes::BatchVotesPublisher;
 use self::batch_votes::BatchVotesWatch;
 use crate::{gossip::ValidatorAddrsWatch, io, pool::PoolWatch, Config, MeteredStreamStats};
+use attestation_status::AttestationStatusWatch;
 use fetch::RequestItem;
 use std::sync::{atomic::AtomicUsize, Arc};
 pub(crate) use validator_addrs::*;
+use zksync_concurrency::time::Duration;
```

Comment:
nit: please use qualified names for concurrency imports - in this case "use zksync_concurrency::time;"